generatorcore/generator.py

```diff
@@ -32,6 +32,9 @@
 from . import industry2030
 from .lulucf2030 import lulucf2030_pyr
 
```

Comment:
I think this is the only place in the code where we mention einflussbilanz. I think that is unfortunate. For several reasons:

- Our source code is english and this is a term in German
- It is a central topic so we should probably have it more prominently somewhere in the code or failing that in a README in the code
- We make this even harder to guess here by appreviating it with eb